pkg/provision/providers/qemu/create.go

```diff
@@ -100,6 +94,12 @@ func (p *provisioner) Create(ctx context.Context, request provision.ClusterReque
 		return nil, err
 	}
 
+	fmt.Fprintln(options.LogWriter, "creating dhcpd")
```

Comment:
moving this code to after creating nodes intentional?

Reply:
Yep, as qemu creates the bridge interface the dhcpd server is attached to, at least one machine has to be created first

Reply:
I added a comment above the line for extra clarity as it's definitely confusing at the first glance